CHANGELOG.md

```diff
@@ -17,12 +17,56 @@ the APIs in these libraries are stable, and are ready for production use.
 
 ### Updated Libraries
 
+- [AI Platform API](/google/cloud/aiplatform/README.md) - new
+  `NotebookServiceClient` and `PersistentResourceServiceClient`
 - [Cloud Dataplex API](/google/cloud/dataplex/README.md) - new
   `CatalogServiceClient`
 - [Network Services API](/google/cloud/networkservices/README.md) - new
   `DepServiceClient`
-- [AI Platform API](/google/cloud/aiplatform/README.md) - new
-  `NotebookServiceClient` and `PersistentResourceServiceClient`
+- [SecurityCenter](/google/cloud/securitycenter/README.md) - add v2 version of the library
+
+### [Bigtable](/google/cloud/bigtable/README.md)
+
+- fix(bigtable): respect GOOGLE_CLOUD_CPP_OPENTELEMETRY_TRACING ([#13748](https://github.com/googleapis/google-cloud-cpp/pull/13748))
+
+### [Pub/Sub](/google/cloud/pubsub/README.md)
+
+- docs(pubsub): add kinesis sample ([#13593](https://github.com/googleapis/google-cloud-cpp/pull/13593))
+
+### [Spanner](/google/cloud/spanner/README.md)
+
+- feat(spanner): add ExcludeTransactionFromChangeStreamsOption ([#13898](https://github.com/googleapis/google-cloud-cpp/pull/13898))
+- feat(spanner): add spanner::Value support for TypeCode::FLOAT32 ([#13862](https://github.com/googleapis/google-cloud-cpp/pull/13862))
+- feat(generator): add polling policy sample ([#13797](https://github.com/googleapis/google-cloud-cpp/pull/13797))
```

Comment:
Remove (at least from Spanner section?).

Reply:
Oops missed that one. Thanks Brad.

Reply:
It looks like you removed the #13759 entry too (not that that's a huge deal).

Reply:
Fixed